### PR TITLE
return pull request creation response

### PR DESCRIPTION
**Pull request by hipstersmoothie**

### CODE REVIEW

File: lib/create-pull-request.js [hunk 1]
```diff
@@ -71,7 +71,7 @@ async function createPullRequest(
     ref: `refs/heads/${head}`
   })
 
-  response = await octokit.pulls.create({
+  return octokit.pulls.create({
```
Comment (gr2m) [inline lib/create-pull-request.js]: Maybe we should update the README to show how to get the pull request information, maybe something like

```js
const { number } = octokit.createPullRequest({
// ...
```

Comment (gr2m) [reply lib/create-pull-request.js]: oh and can you please add a test so we don’t break that behavior in future? 🙏

**Comment by MattIPv4** [comment]
+10 for this.

File: lib/create-pull-request.js (hunk 1 above)
Comment (MattIPv4) [reply lib/create-pull-request.js]: @gr2m Regarding docs, just link to the pulls.create section of the octokit docs?

Comment (gr2m) [reply lib/create-pull-request.js]: Yes, you can link to that for details, but I would suggest we amend the current usage example at https://github.com/gr2m/octokit-create-pull-request#usage to show that something is returned. As most people will want the pull request number, I’d do `const { number } = octokit.createPullRequest({` as suggested above

**Pull request closed without merging**